tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -322,7 +344,9 @@ public:
    std::unique_ptr<REntry> CreateBareEntry() const;
    REntry *GetDefaultEntry() const;
 
-   RFieldZero *GetFieldZero() const { return fFieldZero.get(); }
+   Detail::RFieldBase::RBulk GenerateBulk(std::string_view fieldName);
```

Comment:
I don't see the immediate connection of `GenerateBulk` to changing `GetFieldZero` - is this required here or did it just happen to end up in the same commit?

Reply:
I think that's right, not needed here.